app/assets/javascripts/discourse/app/templates/components/sidebar.hbs

```diff
@@ -0,0 +1,6 @@
+{{#if @shouldDisplay}}
```

Comment:
Not a blocker by any means, but I wonder whether it would be more practical to have this property toggle a class in the component element. There's pros and cons to this, the main pros being that with the class you can animate the toggling of the sidebar, would be useful especially on mobile.

Reply:
Definitely possible but I'm going to leave this for the future once we have a need for it.